src/server.h

```diff
@@ -1742,10 +1742,10 @@ struct redisServer {
     off_t aof_rewrite_base_size;    /* AOF size on latest startup or rewrite. */
     off_t aof_current_size;         /* AOF current size (Including BASE + INCRs). */
     off_t aof_last_incr_size;       /* The size of the latest incr AOF. */
-    off_t aof_fsync_offset;         /* AOF offset which is already requested to be synced to disk. */
+    off_t aof_fsync_offset;         /* AOF offset which is already requested to be synced to disk.
```

Comment:
maybe we better rename the variable too.
so that if we cherry pick any change related to it to another branch / fork it'll not compile without making additional adjustments.
how about `aof_last_incr_fsync_offset`?

Reply:
sound good, i will modify it later